manifests/charts/istiocoredns/values.yaml

```diff
@@ -44,6 +44,9 @@ istiocoredns:
     targetAverageUtilization: 80
 
 global:
+  podDNSSearchNamespaces:
```

Comment:
Can you sort this alphabetically?

Reply:
sure, let me do that

Reply:
It would be good to add some description for the field and what it does eg: [like this](https://github.com/istio/istio/blob/master/manifests/examples/multicluster/values-istio-multicluster-gateways.yaml#L16).

Reply:
done, waiting for CI

Reply:
done. please check